docker/ssl/Makefile

```
@@ -44,14 +48,36 @@ subjectAltName = @alt_names
+DNS.2 = localhost
+endef
+
 define ANNOUNCE_BODY
 Version $(VERSION) of $(PACKAGE_NAME) has been released.
 
 It can be downloaded from $(DOWNLOAD_URL).
 
 etc, etc.
 endef
-all: clean_certs ca server_cert things_grpc_certs auth_grpc_certs
+all: clean_certs ca server_cert things_grpc_certs auth_grpc_certs agent_certs
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)